src/dev/src/app/app.component.ts

```diff
@@ -3,11 +3,26 @@
  * This software is released under MIT license.
  * The full license information can be found in LICENSE in the root directory of this project.
  */
-import { Component } from '@angular/core';
+import { Component, ViewEncapsulation } from '@angular/core';
 import { Route } from '@angular/router';
 import { APP_ROUTES } from './app.routing';
+import { environment } from '../environments/environment';
 
-@Component({ selector: 'app-root', templateUrl: './app.component.html' })
+@Component({
+  selector: 'app-root',
+  templateUrl: './app.component.html',
+  encapsulation: ViewEncapsulation.None,
+})
 export class AppComponent {
   public routes: Route[] = APP_ROUTES;
+
+  constructor() {
+    if (environment.dark) {
+      // tslint:disable-next-line
+      require('style-loader!./../../../clr-angular/dark-theme.scss');
```

Comment:
I'm just wondering if there is any particular reason this is in the app component with no encapsulation. It feels more like `main.ts`'s job to me. Probably a nitpick.